Solstice (#1629)
withastro/astro.build: src/content/blog/solstice-festival.mdx

---
title: 'Astro Solstice Festival'
description: 'The planets have aligned to bring you the Astro Solstice Festival – a week-long celebration of all things Astro'
homepageLink:
  title: '🌞 Solstice Festival'

>   subtitle: 'Join us live'

Author: ```suggestion
  subtitle: 'Live all week'
```

Join us from **June 23rd to June 27th, 2025** for a series of exciting events, including live streams, community challenges, and more. Whether you're a seasoned Astro developer or just getting started, there's something for everyone.

[See the full schedule below](#schedule) 👇

<Image src={poster} alt="Astro Solstice Festival poster. See below for content" width="768" />

## Schedule

More details will be added throughout the week, so check back often and follow us on all the socials.

### Monday, June 23rd

- **[Introducing Astro Mart](/blog/astro-mart/)**: get your festival swag on, with the launch of [Astro Mart](https://shop.astro.build), our new online store for all things Astro. Try the new Summer 2025 collection for festival-ready sunshine vibes.

### Tuesday, June 24th

- <Mention name="sarah" /> joins the [Scrimba Town Hall](https://www.youtube.com/watch?v=ea8kp0Z-gYk) to talk about her journey from Scrimba student to Astro core maintainer, what a regular day maintaining Astro looks like, and [Scrimba's new introductory Astro course](https://scrimba.com/intro-to-astro-c00ar0fi5u?via=astro).

> - **Learning Astro**: a live panel discussion featuring Chris Pennington and James Q. Quick, moderated by <Mention name="sarah" /> talking about the different ways people learn Astro, tips and tricks for learning, how they got involved producing Astro content, and how they see their roles as teachers in the Astro community.

Author: ```suggestion
- **Learning Astro**: a live panel discussion featuring Astro course creators [Chris Pennington](https://learnastro.dev/?code=ASTRO_PROMO) and [James Q. Quick](https://scrimba.com/intro-to-astro-c00ar0fi5u?via=astro), moderated by <Mention name="sarah" /> talking about the different ways people learn Astro, tips and tricks for learning, how they got involved producing Astro content, and how they see their roles as teachers in the Astro community.
```

> - **Learning Astro**: a live panel discussion featuring Astro course creators [Chris Pennington](https://learnastro.dev/?code=ASTRO_PROMO) and [James Q. Quick](https://scrimba.com/intro-to-astro-c00ar0fi5u?via=astro), moderated by <Mention name="sarah" /> talking about the different ways people learn Astro, tips and tricks for learning, how they got involved producing Astro content, and how they see their roles as teachers in the Astro community.

Reviewer: ```suggestion
- **[Learning Astro](https://youtube.com/live/0mKNSwi7Wvk?feature=share)**: a live panel discussion featuring Astro course creators [Chris Pennington](https://learnastro.dev/?code=ASTRO_PROMO) and [James Q. Quick](https://scrimba.com/intro-to-astro-c00ar0fi5u?via=astro), moderated by <Mention name="sarah" /> talking about the different ways people learn Astro, tips and tricks for learning, how they got involved producing Astro content, and how they see their roles as teachers in the Astro community.
```

### Wednesday, June 25th

> - **Perfect Images**: <Mention name="matt" /> joins the [Coding in Public live stream]() to discuss everything Astro images, including new responsive image features, image optimization, and more.

Author: ```suggestion
- **Perfect Images**: <Mention name="matt" /> joins the [Coding in Public live stream](https://codinginpublic.dev/) to discuss everything Astro images, including new responsive image features, image optimization, and more.
```

Reviewer: I wonder whether his YouTube channel might be a better link here? We can also ask for a stream link when known.

> - **[Perfect Images]((https://codinginpublic.dev/)**: <Mention name="matt" /> joins the [Coding in Public live stream](https://codinginpublic.dev/) to discuss everything Astro images, including new responsive image features, image optimization, and more.

Reviewer: ```suggestion
- **[Perfect Images](https://codinginpublic.dev/)**: <Mention name="matt" /> joins the [Coding in Public live stream](https://codinginpublic.dev/) to discuss everything Astro images, including new responsive image features, image optimization, and more.
```

Reviewer: Maybe one a YouTube link, even just temp,  vs both the same

> - **[Perfect Images](https://codinginpublic.dev/)**: <Mention name="matt" /> joins the [Coding in Public live stream](https://www.youtube.com/@CodinginPublic) to discuss everything Astro images, including new responsive image features, image optimization, and more.

Author: ```suggestion
- **[Perfect Images](https://youtube.com/live/Z6w4G_6k7rw?feature=share)**: <Mention name="matt" /> joins the [Coding in Public live stream](https://youtube.com/live/Z6w4G_6k7rw?feature=share) to discuss everything Astro images, including new responsive image features, image optimization, and more.
```
